ihaskell-display/ihaskell-widgets/ihaskell-widgets.cabal

```diff
@@ -56,24 +56,31 @@ library
   -- Modules included in this library but not exported.
   other-modules:       IHaskell.Display.Widgets.Button
                        IHaskell.Display.Widgets.Image
-                       IHaskell.Display.Widgets.Dropdown
+                       IHaskell.Display.Widgets.Bool.CheckBox
+                       IHaskell.Display.Widgets.Bool.ToggleButton
+                       -- IHaskell.Display.Widgets.Dropdown
```

Comment:
I'll try and get the selection widget working in this PR then. I ran into some issues when trying to implement them in full generality.